components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -580,9 +580,8 @@ define([
             warnMissingDOBUnaff = warnMissingDOBUnaff || !is_affected;
          }
 
-         output += "0\t"; // TODO: Genetic test status
```

Comment:
So the old order was an always `0` column, followed by a `status` column, and the new order is a status column, followed by a testing method column. Was it wrongly implemented in the first place, or is this something that changed between BOADICEA versions? I see that the documentation in https://pluto.srl.cam.ac.uk/bd3/v3/docs/BWA_v3_user_guide.pdf lists them in the original order.

Reply:
Line 625 in the new version:
```
         output += testingMethod + "\t";
         output += mutationStatus + "\t";
```
The order hasn't changed, it's still testing method followed by mutation status.